### ISSUE TITLE
게시판 읽을 경우 관련알림 삭제 포인트러쉬 모듈 등 반영 요청합니다.

### ISSUE THREAD

**Issue by wookho** [xe-public/xe-module-ncenterlite#123]
포인트러쉬 모듈의 경우 지금 많은 사이트에서 사용하는 대표적인 도규먼트기반 모듈입니다. 해당 모듈에서도 대부분 알림센터 기능이 똑같이 동작하지만 게시판 읽을 경우 관련알림 삭제 기능이 한정되어 있어 게시판 이외의 모듈에서는 동작하지 않게 되어 있습니다.

https://github.com/xe-public/xe-module-ncenterlite/blob/develop/ncenterlite.controller.php#L525

dispPointrushContent 를 추가해주셔서 포인트러쉬모듈에서도 게시글을 읽으면 알림이 삭제되어 사이트에서 동일한 동작이 될 수 있게 하면 좋을 것 같습니다.

**Comment by bjrambo** [comment]
포인트 러쉬의 경우 다른 모듈이기 때문에 알림센터에서 제어하기 위해서는 개발자에게 모듈이 있어야 한다는 점이 필요합니다.

구체적으로 같은 document_srl 을 이용하더라도 읽는것에 따라 달라질 수 있다고 생각되어서 그냥 그내용을 그대로 추가할 수 없어보이지만 일딴 해당 if문에 포인트러쉬도 넣어보도록 해볼께요.

**Comment by wookho** [comment]
@bjrambo 네. 감사합니다. 포인트러쉬 모듈은 제가 먼저 코드에 추가해보니 잘 됩니다. 다른 도큐먼트기반의 모듈들도 푸시앱모듈에서 적용해서 문제가 없이 잘 돌아가는 것으로 보아 문제는 없을 듯 합니다. 뭐 이부분은 참고적인 내용이니 그냥 흘려 들어주세요.
